src/sentry/search/events/builder.py

```diff
@@ -169,8 +175,8 @@ def resolve_time_conditions(self) -> None:
         ), "Both start and end params must be datetime objects"
 
         # Strip timezone, which are ignored and assumed UTC to match filtering
-        self.start: datetime = self.params["start"].replace(tzinfo=timezone.utc)
-        self.end: datetime = self.params["end"].replace(tzinfo=timezone.utc)
```

Comment:
Why'd you remove this?

Reply:
I added the definition in `__init__` instead, and mypy was complaining that it these variables were already defined.